flake.nix

```diff
@@ -21,7 +19,7 @@
 
             if pkg.source == registry then
               let
-                sha256 = lockFile'.metadata."checksum ${pkg.name} ${pkg.version} (${registry})";
+                sha256 = pkg.checksum;
```

Comment:
Maybe this should be `pkg.checksum or lockFile'.metadata."checksum ${pkg.name} ${pkg.version} (${registry})"` for compatibility with older lock files?

Reply:
@edolstra fair point, fixed it.